.github/CONTRIBUTING.md

```diff
@@ -85,7 +85,7 @@ to install the dependencies.
 $ git clone https://github.com/Chocobozzz/PeerTube
 $ cd PeerTube
 $ git remote add me git@github.com:YOUR_GITHUB_USERNAME/PeerTube.git
-$ yarn install --pure-lockfile
+$ yarn install --pure-lockfile --production=false
```

Comment:
yarn does install deve dependency by default

Reply:
That's weird... In my case, I haven't mocha installed. I installed it manually, and then it was ts-node (if I remember correctly) that was missing. Then I made `yarn install --pure-lockfile --production=false` and it worked fine.
I have no NODE_ENV var.
I don't understand what happened.

Reply:
You reproduce this bug every time you run `yarn install`?

Reply:
I don't know. You must be right. I suggest to drop the entire PR, and just keep this commit: https://github.com/Chocobozzz/PeerTube/pull/4601/commits/80281efc4dce04552b3f79d9d82abd32e1fc631d

Reply:
Agreed. Done in https://github.com/Chocobozzz/PeerTube/commit/11e520b50d791a0dd48cbb2d0fc681b25eb7cd53